<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>count</class>
 <widget class="QDialog" name="count">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>404</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>统计</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QT.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>330</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-280</x>
     <y>10</y>
     <width>921</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>921</width>
     <height>161</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>921</width>
     <height>161</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:image/count.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="count_card">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>卡号账单查询</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="count_time">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>时间账单查询</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="count_total">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>总营业查询</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../QT.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>count</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>count</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
